addon/-private/environment.js

```diff
@@ -0,0 +1,159 @@
+import Ember from 'ember';
```

Comment:
On reflection, breaking this file apart into a few different modules might help with comprehension (along with some comments) — if you agree, I'll take a pass on that.